ml-agents/mlagents/trainers/components/reward_signals/curiosity/signal.py

```diff
@@ -171,5 +147,5 @@ def _update_batch(
                     feed_dict[self.model.next_visual_in[i]] = _obs
 
         self.has_updated = True
-        run_out = self.policy._execute_model(feed_dict, self.update_dict)
-        return run_out
+        # run_out = self.policy._execute_model(feed_dict, self.update_dict)
```

Comment:
🤔

Reply:
Should probably delete the comment - `feed_dict` is right. I also renamed this method to `prepare_update`.